src/code/z_lifemeter.c

```diff
@@ -442,18 +445,18 @@ void HealthMeter_Draw(GlobalContext* globalCtx) {
                 }
             }
 
-            temp3 = 26.0f + offsetY;
-            temp2 = 30.0f + offsetX;
-            temp4 = 1.0f;
-            temp4 /= 0.68f;
-            temp4 *= 1 << 10;
-            temp1 = 8.0f;
-            temp1 *= 0.68f;
-            gSPTextureRectangle(OVERLAY_DISP++, (s32)((temp2 - temp1) * 4), (s32)((temp3 - temp1) * 4),
-                                (s32)((temp2 + temp1) * 4), (s32)((temp3 + temp1) * 4), G_TX_RENDERTILE, 0, 0,
-                                (s32)temp4, (s32)temp4);
+            heartCenterY = 26.0f + offsetY;
+            heartCenterX = 30.0f + offsetX;
+            heartTexCoord = 1.0f;
+            heartTexCoord /= 0.68f;
+            heartTexCoord *= 1 << 10;
+            halfHeartLen = 8.0f;
```

Comment:
I think we can afford the full word "length"
```suggestion
            halfHeartLength = 8.0f;
```

I also am not a fan of "length" but can't think of a better name